src/wp-admin/js/post.js

```diff
@@ -717,21 +877,29 @@ jQuery(document).ready( function($) {
 			event.preventDefault();
 		});
 
+		// Save the Post Status changes and hide the options.
 		$postStatusSelect.find('.save-post-status').click( function( event ) {
 			$postStatusSelect.slideUp( 'fast' ).siblings( 'a.edit-post-status' ).show().focus();
 			updateText();
 			event.preventDefault();
 		});
 
+		// Cancel Post Status editing and hide the options.
 		$postStatusSelect.find('.cancel-post-status').click( function( event ) {
 			$postStatusSelect.slideUp( 'fast' ).siblings( 'a.edit-post-status' ).show().focus();
 			$('#post_status').val( $('#hidden_post_status').val() );
 			updateText();
 			event.preventDefault();
 		});
-	} // end submitdiv
+	}
 
-	// permalink
+	/**
+	 * Permalink aka slug aka post_name editing
```

Comment:
I think this function needs so more explanation. As far as I can see it is initializing the permalink section, but that could be documented in the JSDoc.